src/PersonObjects/Sleeve/Work/SleeveCrimeWork.ts

```diff
@@ -64,6 +64,7 @@ export class SleeveCrimeWork extends Work {
   APICopy(): Record<string, unknown> {
     return {
       type: this.type,
+      name: this.crimeType
```

Comment:
the player version is called `crimeType` so this should be too.

Reply:
switched